src/condition.rs

```diff
@@ -41,3 +41,37 @@ impl From<u8> for Condition {
         }
     }
 }
+
+// Control bits condition
+pub(crate) enum ModeBits {
+    OldUser = 0x00,
```

Comment:
Binary or Hex representation?

Reply:
For the `condition` and `istruction` (`alu_istruction.rs` && `condition.rs` al the moment) we decided to use the `hex` notation.

But, we can still talk about it.

Reply:
It's ok for me!